src/server/conn.rs

```diff
@@ -134,10 +134,16 @@ impl Http {
         self
     }
 
+
     /// Set the maximum buffer size for the connection.
     ///
     /// Default is ~400kb.
+    /// The minimum value that h1 can handle is enforced and will panic if smaller.
```

Comment:
I'd put this in a `# Panics` section, and include the exact minimum, so that people can see what they need to guard against. Something like:

```
/// # Panics
///
/// The minimum value allowed is 8192. This method panics if the passed `max` is less than the minimum.
```

Reply:
Nice, didn't know about this. Thanks for the guidance!